Library/Homebrew/os/mac/xquartz.rb

```diff
@@ -2,7 +2,7 @@
 
 module OS
   module Mac
-    X11 = XQuartz = Module.new # rubocop:disable Style/MutableConstant
+    X11 = Module.new # rubocop:disable Style/MutableConstant
 
     module XQuartz
```

Comment:
Can also be restructured as:
```suggestion
   XQuartz = Module.new do
```